lms/resources/_js_config/__init__.py

```diff
@@ -413,6 +413,7 @@ def _get_students(self):
             )
             yield {
                 "userid": h_user.userid(self._authority),
+                "lmsId": grading_info.user_id,
```

Comment:
I think here `userid` is just a word/concept vs `lmsId` which is more like `LISResultSourcedId`

Also I think having `userId` and `userid` would be extra confusing so even if the attribute in GradingInfo is called "user_id" I'm exposing it as lmsId which is a name we use in other parts of the modeling.